interfaces/Portalicious/src/app/services/registration-action-menu.service.ts

```diff
@@ -0,0 +1,95 @@
+import { computed, inject, Injectable } from '@angular/core';
+
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { PermissionEnum } from '@121-service/src/user/enum/permission.enum';
+
+import {
+  REGISTRATION_STATUS_ICON,
+  REGISTRATION_STATUS_VERB,
+} from '~/domains/registration/registration.helper';
+import { RegistrationStatusChangeTarget } from '~/domains/registration/registration.model';
+import { AuthService } from '~/services/auth.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RegistrationActionMenuService {
+  private readonly authService = inject(AuthService);
+
+  readonly canChangeStatus = computed(
```

Comment:
Come to think of it, the fact that this is computed makes no sense because it doesn't recompute 😄 Maybe we need to have a conversation about what 'computed' means?